Furball.Engine/Engine/Graphics/Drawables/UiElements/DrawableButton.cs

```diff
@@ -40,11 +40,22 @@ public class DrawableButton : CompositeDrawable {
     /// <summary>
     ///     The outline color of the button
     /// </summary>
-    public Color OutlineColor;
+    public Color OutlineColor {
+        get => this._outlineDrawable.ColorOverride;
+        set => this._outlineDrawable.ColorOverride = value;
+    }
+
+    private Color _buttonColor;
     /// <summary>
     ///     The fill color of the button
     /// </summary>
-    public Color ButtonColor;
+    public Color ButtonColor {
+        get => this._buttonColor;
+        set {
+            this._backgroundDrawable.ColorOverride = value;
+            this._buttonColor = value;
+        }
+    }
```

Comment:
This variable here, `_buttonColor` helps deal with hover effects. Without using it to store the initial button color in this fashion, repeatedly hovering over the button would cause the button's color to "drift" and not reset to normal.